servicetalk-http-netty/src/main/java/io/servicetalk/http/netty/AbstractLifecycleObserverHttpFilter.java

```diff
@@ -161,6 +161,12 @@ public final HttpExecutionStrategy requiredOffloads() {
         return HttpExecutionStrategies.offloadNone();
     }
 
+    @Override
+    public final HttpExecutionStrategy influenceStrategy(final HttpExecutionStrategy strategy) {
```

Comment:
is it necessary to implement both methods? I thought @bondolo set this up so that if `requiredOffloads` is implemented we should take the strategy from there?

Reply:
The goal was to make all existing methods `final` even of they are deprecated. It won't be invoked at runtime